docs/user-manual/docker.adoc

```diff
@@ -0,0 +1,130 @@
+= Docker
+:idprefix:
+:idseparator: -
+
+One of the simplest ways to get started with ActiveMQ Artemis is by using one of our Docker images.
+
+== Official Images
+
+Official https://www.docker.com/[Docker] images are https://hub.docker.com/r/apache/activemq-artemis/tags[available on dockerhub].
+Images are pushed along with all the other distribution artifacts for every release.
+The fastest, simplest way to get started is with this command which will create and start a container named `mycontainer` and remove it when it terminates:
+
+[,console]
+----
+$ docker run --rm --name mycontainer -it -p 61616:61616 -p 8161:8161 apache/activemq-artemis:latest-alpine
```

Comment:
Just a small tweak. I think we should "educate" users to use detached mode, and to use logs to look at the output.

I'm not strong opinionated about this though. just a suggestion.

Reply:
Done.